Gems/AtomLyIntegration/CommonFeatures/Code/Source/PostProcess/ColorGrading/EditorHDRColorGradingComponent.cpp

```diff
@@ -199,7 +199,10 @@ namespace AZ
             }
 
             const char* LutAttachment = "LutOutput";
-            const AZStd::vector<AZStd::string> LutGenerationPassHierarchy{ "LutGenerationPass" };
+            const AZStd::vector<AZStd::string> LutGenerationPassHierarchy{
+                "MainPipeline_0",
```

Comment:
Is there a better way to get the scene's main render pipeline name?

Reply:
You can use this code to get the render pipeline name

            auto atomViewportRequests = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
            auto defaultViewportContext = atomViewportRequests->GetDefaultViewportContext();
            auto renderPipelineName = defaultViewportContext ->GetCurrentPipeline()->GetId();

Reply:
This works! PR moved to: https://github.com/o3de/o3de/pull/6436